packages/now-cli/src/index.js

```diff
@@ -13,7 +13,6 @@ try {
     process.exit(1);
   }
 }
-import 'core-js/modules/es7.symbol.async-iterator';
```

Comment:
This was added 2 years ago, but the dependency no longer exists https://github.com/vercel/vercel/commit/64765b393a4be8d1b6e081d0d29283ec7790ef6f#diff-8adafe1e71c9fbe6d42d1ad18ad6e174R6